hs/DevelopmentConfig.hs

```diff
@@ -46,8 +52,30 @@ genCssPath = "../public/style/"
 cssStyle :: Clay.Config
 cssStyle = Clay.pretty
 
+-- CALENDAR RESPONSE DATES
+
+-- | First Monday of the Fall term.
+firstMondayFall :: Day
+firstMondayFall = fromGregorian 2015 09 14
+
+-- | Last Wednesday of the Fall term.
+lastWednesdayFall :: Day
+lastWednesdayFall = fromGregorian 2015 12 02
+
+-- | First Monday of the Winter term.
+firstMondayWinter :: Day
+firstMondayWinter = fromGregorian 2016 01 11
+
+-- | Last Monday of the Winter term.
+lastMondayWinter :: Day
+lastMondayWinter = fromGregorian 2016 04 04
+
+-- | Out of date day.
```

Comment:
'OutDay' is used to control forbidden inputs for days in the 'getDayFall' and 'getDayWinter' functions in CalendarResponse.hs. I will include more precise comments on this. Thank you.